testdata/p4_16_samples_outputs/table-entries-priority-bmv2.p4.entries.txt

```diff
@@ -0,0 +1,75 @@
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33556639
+      match {
+        field_id: 1
+        ternary {
+          value: "\021\021"
+          mask: "\000\017"
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\001"
+          }
+        }
+      }
+      priority: 1
```

Comment:
@priority annotation in source program appears to be ignored here.  I thought the P4_16 language spec didn't even support explicit priority here -- I thought it went with "entries always in source code order" for ternary tables.  Should this test be removed?  If not, it definitely seems wrong in priority values, at least.